lms/envs/common.py

```diff
@@ -1905,6 +1905,12 @@
     'ROOT_PATH': '/tmp/edx-s3/grades',
 }
 
+FINANCIAL_REPORTS = {
+    'STORAGE_TYPE': 'localfs',
+    'BUCKET': 'edx-financial-reports',
+    'ROOT_PATH': '/tmp/edx-s3/financial_reports',
```

Comment:
I will schedule a JIRA task for dev ops to create a new bucket and to set the appropriate config.